### PR TITLE
Fix/error out of merge/no docker daemon

### PR DESCRIPTION
**Pull request by fpdotmonkey**
if you run `./runtests.sh` while the docker daemon is not running, the program should exit and notify you that you should have the docker daemon running.

### CODE REVIEW

File: runtests.sh [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
-unittest_exit_code=$?
-deactivate
-
-if [ $behave_exit_code -ne 0 ]; then
-    echo "behave tests failed!"
-fi
-if [ $unittest_exit_code -ne 0 ]; then
-    echo "unittests tests failed!"
-fi
-exit $(($behave_exit_code + $unittest_exit_code))
```
Comment (capsulecorplab) [inline runtests.sh:18]: Line 3, 6, & 9-18 were just added in PR #7. We probably don't want to remove them?

Comment (fpdotmonkey) [reply runtests.sh:18]: They were added in #7 but removed in favor of the code from #23

Comment (fpdotmonkey) [reply runtests.sh:18]: They were removed in favor of the code from #23

**Comment by capsulecorplab** [comment]
Closing PR, since there appears to be zero file changes.

**Pull request closed without merging**